src/test/java/com/google/firebase/auth/multitenancy/FirebaseTenantClientTest.java

```diff
@@ -312,6 +321,20 @@ public void testDeleteTenantWithNotFoundError() {
     checkUrl(interceptor, "DELETE", TENANTS_BASE_URL + "/UNKNOWN");
   }
 
+  @Test
```

Comment:
Ideally we need a way to test all APIs in the class with the emulator.

Reply:
Agreed. See the previous comment.